earth_enterprise/integration_test/test_teardown.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ASSETROOT=/gevol/assets
+
+sudo service gefusion stop
```

Comment:
This script doesn't ensure there are no Fusion entries referring to the files being deleted below.

For example, there are resources above that seem to imply they have Points of Interest data, which would be inserted in PostgreSQL tables.  It's true that after deleting these files there may be no resources that refer to those entries, and, therefore, this test would be creating orphaned SQL data.

If that is not the case, it should be noted here.  If it is the case, it should be fixed, if that's easy, or noted here, and another ticket should be created to fix the build up of orphaned data.

Reply:
Fusion doesn't use postgres based on the fact that postgres is not required for Fusion. Based on the section "Clean up the tutorial work space" in this: http://www.opengee.org/geedocs/answer/6028272.html, I think we can say that everything that fusion creates is stored under the asset root. Do you still think there is a need for a comment here?

Reply:
Ah, yes, you're right. Those Fusion projects are never published, so there wouldn't be any of that data.

You could include a link to that document, but I'm fine even without a comment.